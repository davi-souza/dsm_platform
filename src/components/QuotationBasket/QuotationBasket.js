import React from 'react';
import Card from '@material-ui/core/Card';
import CardContent from '@material-ui/core/CardContent';
import CardActions from '@material-ui/core/CardActions';
import Button from '@material-ui/core/Button';
import './QuotationBasket.scss';

function QuotationBasket() {
	return (
		<Card className="quotation-basket">
			<CardContent></CardContent>
			<CardActions>
				<Button>ENCOMENDAR</Button>
			</CardActions>
		</Card>	
	);
}

export default QuotationBasket;
