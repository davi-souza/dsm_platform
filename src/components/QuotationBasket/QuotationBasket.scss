.quotation-basket {
	position: sticky;
	top: 1rem;
	left: 0;
}
