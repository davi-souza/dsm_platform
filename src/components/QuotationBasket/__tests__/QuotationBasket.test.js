import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import QuotationBasket from '../QuotationBasket';

configure({ adapter: new Adapter() });

describe('Quotation basket tests', () => {
	it('should render without crash', () => {
		const component = shallow(<QuotationBasket />);
		expect(component).toMatchSnapshot();
	});
});
