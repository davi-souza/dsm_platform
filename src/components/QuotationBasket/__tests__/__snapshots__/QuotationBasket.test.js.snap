// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quotation basket tests should render without crash 1`] = `ShallowWrapper {}`;
